src/Core/Manifest/ClassManifestErrorHandlerTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Core\Manifest\ClassManifestErrorHandler;
+use PhpParser\Error;
+
+class ClassManifestErrorHandlerTest extends SapphireTest
```

Comment:
PHPCS is pointing out this class is missing a namespace

Reply:
Fixed, thanks PHPCS! :)